### PR TITLE
SCRUM-2252 Added check for string on bioEntity fields

### PR DESCRIPTION
**Pull request by adamgibs**

### CODE REVIEW

File: src/main/cliapp/src/utils/utils.js
Comment (markquintontulloch) [inline src/main/cliapp/src/utils/utils.js]: We should also add "assertedAllele" to this.  Entering text into the "with" and "assertedGenes" fields doesn't throw an error but just removes the text on saving - I don't know if we can deal with those fields in the same way due to the multi-select nature of those columns, but the current behaviour may be confusing (probably something for a separate ticket though)

**Review by markquintontulloch — CHANGES_REQUESTED** [review]
Apart from the addition of "assertedAllele" to the list of fields to check, this looks good to me

File: src/main/cliapp/src/utils/utils.js
Comment (adamgibs) [reply src/main/cliapp/src/utils/utils.js]: @markquintontulloch Ah, okay. I'll added assertedAllele. The multi-value fields of all kinds can't submit string types since the only value that they track is the array of values. My think was that adding a string check for the multi-value fields was out of the scope of this ticket, but maybe Chris will think differently.

Comment (markquintontulloch) [reply src/main/cliapp/src/utils/utils.js]: I agree about it being outside the scope of this ticket, as the ticket was to address the fields that throw errors

Comment (adamgibs) [reply src/main/cliapp/src/utils/utils.js]: @markquintontulloch "assertedAllele" has been added to the validation list

**Review by markquintontulloch — APPROVED** [review]

**Comment by markquintontulloch** [comment]
@adamgibs might be worth holding off on merging this until Ketaki's code has been merged - the uncaught errors that are fixed by this are a good test case for her stuff

**Comment by adamgibs** [comment]
@markquintontulloch  Got it. Will do.

**Pull request merged**
